python/lsst/ap/verify/ingestion.py

```diff
@@ -48,6 +48,8 @@
 from lsst.pipe.tasks.ingestCalibs import IngestCalibsTask
 from lsst.pipe.tasks.ingestCuratedCalibs import IngestCuratedCalibsTask
 
+_LOG = lsst.log.Log.getLogger(__name__)
```

Comment:
Do we need this boilerplate in every module? I'm a bit suspicious of global variables...

Reply:
Where else would you put it? We use this form all the time in middleware.